File: 75-sort-colors/sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        one=0
        two=len(nums)-1
        zero=0
        while(one<=two):
            if nums[one]==1:
                one+=1
            elif nums[one]==0:
                t=nums[one]
                nums[one]=nums[zero]
                nums[zero]=t
                one+=1
                zero+=1
            else:
                l=nums[one]
                nums[one]=nums[two]
                nums[two]=l
                two-=1
        

File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int zero=0,one=0,two=nums.size()-1;
       while(one<=two){
            if(nums[one]==1){
                one++;
            }else if(nums[one]==0){
                int t=nums[zero];
                nums[zero]=nums[one];
                nums[one]=t;
                zero++;
                one++;
            }else{
                int t=nums[two];
                nums[two]=nums[one];
                nums[one]=t;
                two--;
            }
       }
    }
};